pkg/drivers/generic/generic.go

```diff
@@ -135,6 +135,10 @@ func (d *Generic) Migrate(ctx context.Context) {
 }
 
 func configureConnectionPooling(connPoolConfig ConnectionPoolConfig, db *sql.DB) {
+	if connPoolConfig.MaxIdle < 2 {
```

Comment:
Would it be possible to make this number a const with a descriptive name that might include the unit of measure, ie: `maxIdeSeconds = 2` or something appropriate?

Reply:
I think that I actually see a better way to handle this, inspired by upstream:
https://github.com/golang/go/blob/release-branch.go1.13/src/database/sql/sql.go#L427

Reply:
Still requires the reader to have seen it first to reference back or know to check up there. This is considered a "magic number" and generally should be avoided.